docs/4.x/tracking-api.md

```diff
@@ -43,6 +43,7 @@ _(We recommend that these parameters be used if the information is available and
 * `cookie`  &mdash; when set to 1, the visitor's client is known to support cookies.
 
 * `ua` &mdash; An override value for the **User-Agent** HTTP header field. The user agent is used to detect the operating system and browser used.
+* `uadata` &mdash; JSON encoded **Client Hints** collected by javascript. This will be used to enrich the detection with user agent. (requires Matomo 4.12.0)
```

Comment:
Maybe this is a bit clearer?
```suggestion
* `uadata` &mdash; JSON encoded **Client Hints** collected by javascript. This will be used to enrich the detected user agent data. (requires Matomo 4.12.0)
```